qgis2to3/codeporting/qgis2to3.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+import sys, os
+import functools
+import qgis2to3.codeporting
+
+import lib2to3.main
+
+import lib2to3.refactor
+
+lib2to3.refactor._open_with_encoding = functools.partial(open, newline='')
+
+def main():
+    sys.path.append(os.path.abspath(os.path.dirname(qgis2to3.codeporting.__file__)))
```

Comment:
Is the path mangling here required?

Reply:
Unsure it was just there before but maybe because it was a script.  I will try without it.